ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/CardinalClient.kt

```diff
@@ -109,6 +109,9 @@ internal class CardinalClient {
             cardinalConfigurationParameters.uiCustomization = it.cardinalUiCustomization
         }
 
+        request.requestorAppUrl?.let {
```

Comment:
Could we add a unit tests for this new condition?

Reply:
Ah, I had assumed this wasn't testable on Android since it isn't on iOS. But it looks like I was wrong, test added here: https://github.com/braintree/braintree_android/pull/1232/commits/d9ce940f69b7fa30d894e882aa4076b2dd7b1644